test/evergreen.yml

```diff
@@ -4191,6 +4191,28 @@ tasks:
           test-name: modularity_metrics
           collection: CodeModularityMetrics
 
+  - name: tsan-warning-metrics
+    commands:
+      - command: timeout.update
+        params:
+          exec_timeout_secs: 43200
+          timeout_secs: 43200
```

Comment:
If I get it right, it's 12 hours, if we don't want to spend more than 3 hours for this, should we decrease it?

Reply:
We don't want to spend 3 hours per type of test we run. Overall the task shouldn't last 12 hours

Reply:
Maybe now when we run examples only, we can reduce this limit?